src/common/modals/InstanceExport/ExportPackModal.module.css

```diff
@@ -0,0 +1,34 @@
+.container {
+    height: 85%;
+    width: 100%;
+    /* font-family: 'GlacialIndifferenceRegular'; */
+    padding: 20px;
+    overflow-y: auto;
+  }
+  
+  .selectFolderBtn {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5447870)